signer/src/transaction_coordinator.rs

```diff
@@ -676,54 +678,65 @@ where
     async fn construct_and_sign_stacks_sbtc_response_transactions(
         &mut self,
         chain_tip: &model::BitcoinBlockRef,
+        wallet: &SignerWallet,
+        bitcoin_aggregate_key: &PublicKey,
+    ) -> Result<(), Error> {
+        let fut = self.construct_and_sign_stacks_sbtc_depost_response_transactions(
```

Comment:
```suggestion
        let fut = self.construct_and_sign_stacks_sbtc_deposit_response_transactions(
```

Reply:
Updated in https://github.com/stacks-network/sbtc/pull/1403/commits/81cd5e3993bafb84b920eb1654f0a3cdb22adcd5. I also removed the `sbtc` from the name and did the same in the others in my line of sight. It seems redundant/uninformative here, since everything is sbtc related.

Reply:
Yeah I agree, this names in this file have bugged me for a while 😅